src/i18n/MailboxTranslator.js

```diff
@@ -46,6 +46,11 @@ const translateSpecial = folder => {
 			return t('mail', 'Inbox')
 		}
 	}
+	if (folder.specialUse.includes('priority')) {
+		if (folder.isPriorityInbox === true) {
```

Comment:
Same as L89?

Reply:
Sorry, i didnt get this one? What do you mean?

Reply:
Oh, sorry :) I meant that `=== true` isn't needed, because `if (folder.isPriorityInbox)` checks truthiness